libraries/postgres/src/Database.js

```diff
@@ -642,6 +649,57 @@ class Database {
       this.monitor.log.dbPoolCounts(fields);
     }
   }
+
+  /**
+   * Takes any bytes value (you must ensure this as the caller, e.g. `Buffer.from()`)
+   * and returns an encrypted value for storing in the db with the current crypto key
```

Comment:
It's probably worth an assert that this is an instance of Buffer

Reply:
It looks like `cipher.update` takes either Buffer or string (and also TypedArray and DataView ?) so I'll assert it is one of those rather than just a Buffer.

Reply:
Actually, forget it. Strings are weird, buffers rule